x-pack/plugins/observability_solution/investigate/common/types.ts

```diff
@@ -32,12 +29,19 @@ export enum InvestigateWidgetColumnSpan {
   Four = 4,
 }
 
-export interface InvestigateTimeline {
+export interface InvestigationRevision {
   id: string;
-  title: string;
-  '@timestamp': number;
-  user: InvestigateUser;
   items: InvestigateWidget[];
+  parameters: GlobalWidgetParameters;
+}
+
+export interface Investigation {
+  id: string;
+  '@timestamp': number;
+  user: AuthenticatedUser;
```

Comment:
Once we have the concept of more users participating in one investigation, do we consider having here an array of authenticated users? Or possibly we could have an extra field invitedUsers. This is definitely not part of version 1, but I am trying to think about what we might introduce in the future that could shape the Investigation interface.